Source/Blazorise/Styles/_layout.scss

```diff
@@ -31,7 +31,7 @@
     color: rgba(0, 0, 0, 0.65);
     /*background: #001529;*/
     /*line-height: 64px;*/
-    /*&-fixed {
+    &-fixed {
```

Comment:
@stsrki this is the change I am referring to that I added back.
It looked like you mistakenly commented this line when you were removing `line-height`.

Without this, there was no more `.b-layout-header-fixed` generated in the CSS, which broke the `fixed` header.

Reply:
Yes you're right, sorry about that. Since it's now back I can proceed with merge.

Reply:
Great! Sounds good to me.
Thanks again.